dev/reference/sql/statements/recover-table.md

```diff
@@ -0,0 +1,107 @@
+---
+title: RECOVER TABLE
+summary: An overview of the usage of RECOVER TABLE for the TiDB database.
+category: reference
+---
+
+# RECOVER TABLE
+
+`RECOVER TABLE` is used to recover the deleted table and the data on it within the GC life time after the `DROP TABLE` statement is executed.
+
+## Syntax
+
+{{< copyable "sql" >}}
+
+```sql
+RECOVER TABLE table_name
+```
+
+{{< copyable "sql" >}}
+
+```sql
+RECOVER TABLE BY JOB ddl_job_id
+```
+
+> **Note:**
+>
+> + `RECOVER TABLE` can not be used to recover the table beyond the GC life time. In this attempt, an error is returned: `snapshot is older than GC safe point 2019-07-10 13:45:57 +0800 CST`.
```

Comment:
"Beyond" is not accurate because it indicates the span of time that's out of the range.  And it's not the table that is out of GC lifetime. A table is deleted, and then there is this countdown time before the storage is re-claimed.